pom.xml

```diff
@@ -175,7 +175,6 @@
                     <groupId>net.java.dev.jets3t</groupId>
                 </exclusion>
             </exclusions>
-            <scope>provided</scope>
```

Comment:
I would make this line separate commit.

...and then let's discuss whether it should be included.  I guess the idea is that spark is not included in the fatjar. I'm not sure whether this change breaks that?

Reply:
It would be cool if we can achieve both, being able to run locally from IDE, and building fat-jar without spark. Maybe removing the `<scope>` achieves exactly that, but I'm not sure.